test/DSE.Open.Observations.Tests/MeasureTests.cs

```diff
@@ -13,23 +12,23 @@ public sealed class MeasureTests
     [Fact]
     public void CanSerializeAndDeserialize_Binary()
     {
-        var measure = new Measure<bool>(s_measureUri, MeasurementLevel.Binary, "Test measure", "[subject] does something");
+        var measure = new Measure<Binary>(s_measureUri, MeasurementLevel.Binary, "Test measure", "[subject] does something");
         AssertJson.Roundtrip(measure);
```

Comment:
Not specific to this PR, but we can probably drop the reflection-based tests and rely on testing with the source generator.